templates/nsq/0/docker-compose.yml.tpl

```diff
@@ -0,0 +1,43 @@
+nsqlookupd01:
+  image: nsqio/nsq
+  command: /nsqlookupd
+nsqlookupd02:
+  image: nsqio/nsq
```

Comment:
Don't use generic docker image tags implicit neither explicit, not reproducible. Please, use a concrete docker image tag.
```
-  image: nsqio/nsq
+  image: nsqio/nsq:v1.0.0-compat
```